src/libs/auth.js

```diff
@@ -110,24 +131,26 @@ export const initializeAuth = _.memoize(async () => {
         fenceStatus: isSignedIn ? state.fenceStatus : {},
         // Load whether a user has input a cookie acceptance in a previous session on this system,
         // or whether they input cookie acceptance previously in this session
-        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(user.getId(), cookiesAcceptedKey) : undefined,
+        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(userId, cookiesAcceptedKey) : undefined,
         isTimeoutEnabled: isSignedIn ? state.isTimeoutEnabled : undefined,
         user: {
-          token: authResponse ? authResponse.access_token : undefined,
-          id: user.getId() || undefined,
+          token: user?.access_token || undefined,
+          id: userId || undefined,
           ...(profile ? {
-            email: profile.getEmail(),
-            name: profile.getName(),
-            givenName: profile.getGivenName(),
-            familyName: profile.getFamilyName(),
-            imageUrl: profile.getImageUrl()
+            email: profile.email,
+            name: profile.name,
+            givenName: profile.givenName,
+            familyName: profile.familyName,
+            imageUrl: profile.picture
           } : {})
         }
       }
     })
   }
-  processUser(getAuthInstance().currentUser.get())
-  getAuthInstance().currentUser.listen(processUser)
+  const userManager = new UserManager(getOidcConfig())
```

Comment:
Instantiate a `UserManager` directly to bootstrap the initial auth -- but install the listeners on the auth instance from `authStore`.

Reply:
Why is the `OidcConfig` needed both here and in the `Main.js` call to `AuthProvider`?